pom.xml

```diff
@@ -51,12 +51,13 @@
 			<version>3.5.0</version>
 			<scope>provided</scope>
 		</dependency>
-		<dependency>
-			<groupId>org.kitteh</groupId>
-			<artifactId>VanishNoPacket</artifactId>
-			<version>3.20</version>
-			<scope>provided</scope>
-		</dependency>
+        <dependency>
+            <groupId>org.kitteh</groupId>
+            <artifactId>VanishNoPacket</artifactId>
+            <version>3.20</version>
+            <scope>system</scope>
+            <systemPath>C:\Users\Max\Desktop\civ plugins\Jars for dependencies\VanishNoPacket.jar</systemPath>
```

Comment:
This won't work buddy. Just use install:install-file for deps like this that aren't hosted.